.github/workflows/test_code.yml

```diff
@@ -10,10 +10,12 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: '3.6'
+      - name: install mypy
+        run: pip3 install mypy==0.770
```

Comment:
it may be a good idea to add here `pydantic` and `fastapi` as they provide mypy types in their modules (which will probably come up as a problem later down the development)